styles/styles.go

```diff
@@ -58,13 +59,20 @@ func SelectTheme(theme string) glamour.TermRendererOption {
 	case "notty":
 		return glamour.WithStyles(glamour.NoTTYStyleConfig)
 	default:
+		bytes, err := os.ReadFile(theme)
+		if err == nil {
+			return glamour.WithStylesFromJSONBytes(bytes)
+		}
+		// Should log a warning so the user knows we failed to read their theme file
```

Comment:
👍  to this, I'm thinking about writing all error messages to a log file because we fail silently in most cases